src/LibVLCSharp/LibVLCSharp.csproj

```diff
@@ -29,7 +29,7 @@ If you need Xamarin.Forms support, see LibVLCSharp.Forms.
 
 LibVLC needs to be installed separately, see VideoLAN.LibVLC.* packages.</Description>
     <TargetFrameworks>netstandard2.1;netstandard2.0;netstandard1.1</TargetFrameworks>
-    <TargetFrameworks Condition="!$([MSBuild]::IsOsPlatform('Linux'))">$(TargetFrameworks);MonoAndroid81;Xamarin.iOS10;Xamarin.Mac20;Xamarin.TVOS10</TargetFrameworks>
+    <TargetFrameworks Condition="!$([MSBuild]::IsOsPlatform('Linux'))">$(TargetFrameworks);MonoAndroid9.0;Xamarin.iOS10;Xamarin.Mac20;Xamarin.TVOS10</TargetFrameworks>
```

Comment:
`MonoAndroid81` won't be enough? Maybe we should provide both `MonoAndroid81` and `MonoAndroid9.0`?

Reply:
Sorry, Mono 81 will be enough,  I went and reverted everything else but must have manually downgraded that version I will switch it.

Reply:
Ok